store/interfaces.json

```diff
@@ -1,5 +1,43 @@
 {
   "footer": {
-    "component": "index"
+    "allowed": [
+      "footer-layout"
+    ],
+    "component": "Footer"
+  },
+  "footer-layout": {
+    "component": "FooterLayout",
+    "composition": "children",
+    "allowed": [
+      "footer-row",
+      "footer-spacer",
+      "social-networks",
+      "accepted-payment-methods",
+      "logo",
+      "newsletter",
+      "unstable--menu",
```

Comment:
```suggestion
      "menu",
```

Reply:
I can't right now, since this is not published yet:
https://github.com/vtex-apps/menu/pull/35

> error:   App build failed with message: I couldn't find an interface "menu". You need to declare it or depend on an app that declares it.